src/main/resources/schema/app-manifest/v1.0/schema.json

```diff
@@ -0,0 +1,254 @@
+{
```

Comment:
A less general name can be given e.g. remote-plugin-manifest.json or plugin-manifest.json

Reply:
This reflects the current repository structure: https://github.com/reportportal/schema-registry/tree/main/docs/app-manifest/v1.0

Reply:
Change to manifest.schema.json